api/src/pcapi/core/users/api.py

```diff
@@ -1316,3 +1316,54 @@ def delete_old_trusted_devices() -> None:
 
     users_models.TrustedDevice.query.filter(users_models.TrustedDevice.dateCreated <= five_years_ago).delete()
     db.session.commit()
+
+
+def _get_users_with_suspended_account_to_notify(expiration_delta_in_days: int) -> list[models.User]:
+    start = datetime.date.today() - datetime.timedelta(days=expiration_delta_in_days)
+
+    # distinct keeps the first row if duplicates are found. Since rows
+    # are ordered by userId and eventDate, this query will fetch the
+    # latest event for each userId.
+    user_ids_and_latest_action = (
+        users_models.User.query.distinct(history_models.ActionHistory.userId)
+        .join(users_models.User.action_history)
+        .filter(
+            sa.or_(
+                history_models.ActionHistory.actionType == history_models.ActionType.USER_SUSPENDED,
+                history_models.ActionHistory.actionType == history_models.ActionType.USER_UNSUSPENDED,
```

Comment:
I want to delete this line but I don't know why it was in the function I cloned in the first place do you think there is a reason why I should not delete it?